examples/automerge-repo-demo-counter-svelte/package.json

```diff
@@ -15,15 +15,16 @@
     "@automerge/automerge-repo-network-websocket": "^1.0.3",
     "@automerge/automerge-repo-storage-indexeddb": "^1.0.3",
     "@automerge/automerge-repo-svelte-store": "^1.0.3",
-    "svelte": "^3.49.0"
+    "svelte": "^3.0.0"
```

Comment:
test with the lowest version specified as the peer dependency in `automerge-repo-svelte-store`